packages/clis/parser/game-files/map-files-parser.ts

```diff
@@ -925,6 +925,25 @@ function postProcess(
     'roads possibly split by terrains, buildings, or curves',
   );
 
+  // Augment mileage targets from defs with position info from sectors.
+  for (const [token, target] of defData.mileageTargets) {
+    if ((target.easting && target.southing) || !target.nodeUid) {
+      continue;
+    }
+    const nodeUid = BigInt(target.nodeUid);
+    const node = nodesByUid.get(nodeUid);
+    if (node) {
+      defData.mileageTargets.set(token, {
+        ...target,
```

Comment:
Ahyes, well spotted.

I wrote it like this intentionally because my original goal was to be able to re-create [mileage-targets.csv](https://github.com/nautofon/ats-towns/blob/california/mileage-targets/mileage-targets.csv). But you're quite right: If we already have coordinates, `nodeUid` is entirely unnecessary, so it shouldn't be in the output file.